gloo/broadcast_one_to_all.h

```diff
@@ -70,14 +44,21 @@ class BroadcastOneToAll : public Algorithm {
       return;
     }
 
+    auto clearToSendBuffer = context_->createUnboundBuffer(nullptr, 0);
```

Comment:
N00b question why this is even needed? Is it because we need to wait for the receiver to be ready?